src/plasma/plasma_manager.cc

```diff
@@ -682,6 +682,9 @@ void process_data_chunk(event_loop *loop,
   int err = read_object_chunk(conn, buf);
   auto plasma_conn = conn->manager_state->plasma_conn;
   if (err != 0) {
+    /** Remove the object from the receives_in_progress set so that
```

Comment:
This comment style should only be used for structs/functions where the comment is intended to be parsed by doxygen to autogenerate documentation. Instead, use

```
/*
 * ... */
```

or

```
//
//
```

we're switching to the latter, but either is ok for now.

Reply:
This applies to other places as well.